ODF 1.3 CSD03 deliverables

svanteschubert opened the pull request.

mistmist commented on src/test/resources/odf1.3/tools/odf2html/export/xhtml/header.xsl:
```diff
@@ -180,7 +180,7 @@
 <xsl:text> }
     </xsl:text>
-        </xsl:if>        
+        </xsl:if>
```
this is just whitespace change, not what the commit message promised?

franciscave commented on src/test/resources/odf1.3/tools/odf2html/export/xhtml/body.xsl:
```diff
@@ -2037,7 +2037,12 @@
                                             <xsl:text>;min-width:</xsl:text>
-                                            <xsl:value-of select="$listLabelWidth"/>
+                                            <xsl:choose>
+                                                <xsl:when test="$listLabelWidth and not($listLabelWidth!='') and not($listLabelWidth!='NaN')">
```
My reading of the XSLT is that the double negation is NOT correct. The aim is only to output the value of $listLabelWidth when it is a valid number, i.e. when both not($listLabelWidth='') and not($listLabelWidth='NaN') are true.

svanteschubert replied:
Seems, when I write a comment and want to resolve the conversation "Resolve conversation" will not overtake the comment. 
I fully agree with both of you, I have thanked Michael for the spotting, I spotted and additional error, I will fix.
Might be good that the HTML output will be indented (for ODF 1.4) as well for review purposes - I did so manually and realized that the fix will add indent to lists. So I will push an update (XSLT and HTML) to this pull request and create a new ZIP with updated HTML.

franciscave reviewed (commented):
Should all dates be '31 August 2020'? If so, they need to be changed in many files, because these commits specify '24 August 2020' in many places.

svanteschubert commented on src/main/resources/odf1.3/OpenDocument-v1.3-dsig-schema.rng:
```diff
@@ -7,17 +7,10 @@
-        AS-IS, WITHOUT ANY IMPLIED OR EXPRESS WARRANTY; there is no warranty of MERCHANTABILITY,
-        FITNESS FOR A PARTICULAR PURPOSE or NONINFRINGEMENT of the rights of others.
-	
+        Open Document Format for Office Applications (OpenDocument) Version 1.3
```
Oh, Paul sent an email to Patrick & me, but not on the list, we will have to double-check with him if this is okay if the link above serves the former issue: https://issues.oasis-open.org/browse/OFFICE-2153

svanteschubert replied:
I have explicitly requested OASIS review for this 👍 
https://lists.oasis-open.org/archives/office/202008/msg00052.html

svanteschubert commented:
> Should all dates be '31 August 2020'? If so, they need to be changed in many files, because these commits specify '24 August 2020' in many places.
> 
> Oh, I think they're updated to '31 August 2020' by a later commit, so probably all good.

Yes, I am usually following the commits due to their additional semantic, but stumble over the same problem of sequential DIFF (where an intermediate state is later exchanged). 
If you have doubt, quickly see all changes of the pull request files via files: https://github.com/oasis-tcs/odf-tc/pull/26/files

franciscave commented:
I think I am seeing unchanged dates '24 August 2020' in XML comments at the top of the .rng and .owl files at least. Maybe these don't matter, but probably best if they match the metadata.

svanteschubert commented:
I have updated the body.xsl and the resulting HTML.
In the future, I leave it up to the reviewer to close a conversation. Sorry, still learning myself! ;-)

svanteschubert commented:
> I think I am seeing unchanged dates '24 August 2020' in XML comments at the top of the .rng and .owl files at least. Maybe these don't matter, but probably best if they match the metadata.

Good spotting, Francis! I have fixed this right away!
This is exactly the reason, why I would love to have variables in all the places and only one configuration file to change the dates ;-)

franciscave commented:
The dates are fixed in 'docs', but not in 'src' I think. Is that correct? For example, compare 'docs/odf1.3/csd03/OpenDocument-v1.3-csd03-manifest-schema.rng' with 'src/test/resources/odf1.3/tools/rng2html/OpenDocument-v1.3-csd03-manifest-schema.html'.

svanteschubert commented:
> The dates are fixed in 'docs', but not in 'src' I think. Is that correct? For example, compare 'docs/odf1.3/csd03/OpenDocument-v1.3-csd03-manifest-schema.rng' with 'src/test/resources/odf1.3/tools/rng2html/OpenDocument-v1.3-csd03-manifest-schema.html'.

So true, I have fixed and added it, had been too focused on the ZIP. Did I mention that I dislike data duplication? ;-) A good Github repo goal for ODF 1.4!

mistmist commented on src/main/resources/odf1.3/OpenDocument-v1.3-part1-introduction_odt/styles.xml:
```diff
@@ -928,13 +928,13 @@
     <style:list-level-properties text:min-label-width="0.762cm" text:min-label-distance="0.127cm"/>
    </text:list-level-style-number>
-   <text:list-level-style-number text:level="2" text:style-name="Zeichenformat" style:num-suffix=" " style:num-format="1" text:display-levels="2">
+   <text:list-level-style-number text:level="2" text:style-name="Zeichenformat" style:num-prefix="Appendix " style:num-suffix="  " style:num-format="1" text:display-levels="2">
```
i wondered what this does but apparently the prefix only applied once to the whole number so it's fine; was somehow thinking it would be applied per-level...
